_drafts/post-mortems-intro.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Post-mortems intro
+author: Ines Martins
+date: 2022-02-01 11:33:00 +0800
+categories: []
+tags: [postmortem]
+math: true
+mermaid: true
+image: /assets/images/posts/20190808/mockup.png
+src: /assets/images/posts/20190808/mockup.png
+width: 800
+height: 500
+---
+
+<h2 data-toc-skip>Background</h2>
+
+In the past, one of the Beyond engineering core values was “We Ship It!” More recently, we translated it to "Ship Early." At Beyond, the best way to ship safely is to ship often, and the best way to ship often is to ship smaller. 
+
+Shipping early is a good way to get feedback, to quickly iterate and improve on code, break wrong assumptions, rearrange logic and fix unexpected (hopefully) small errors…
+
+We work towards perfection, but we realistically know we will never be 100% perfect, not all the time. Releases do not always follow the happy path; something unexpected may happen and then we honor the other Beyond engineering principle, "We Take Ownership." We jump into action and quickly revert or fix those errors.
+
+![Desktop View](/assets/images/posts/post-mortems-intro/ship-it.gif){: width="400" height="200" }
+_“What went wrong, and how/what do we learn from it?”_
+
+If we take notes during the “fixing process,” from the time we found out something was wrong and what we have done to make it right, we can build a post-mortem!
+
+The concept of “post-mortem” originates from medical jargon. By definition “it is a surgical [**examination of a dead body in order to find out the cause of death**](https://dictionary.cambridge.org/pt/dicionario/ingles/order).” This analogy can be transcribed to the engineering field as a detailed examination [**of an incident after it has happened**](https://dictionary.cambridge.org/pt/dicionario/ingles/happen). It is a useful tool to describe major incidents for the rest of the team to have a clearer picture of what went wrong and what was done to fix it!
```

Comment:
what is this https://dictionary.cambridge.org/pt/dicionario/ingles/order & https://dictionary.cambridge.org/pt/dicionario/ingles/happen, i think we can remove them lol